config_gen.py

```diff
@@ -252,11 +252,19 @@ def run(cmd, *args, **kwargs):
 
     elif(pro_files):
         # qmake
-        # make sure there is only one .pro file
         if len(pro_files) != 1:
-            print("ERROR: Found {} .pro files (expected one): {}.".format(
-                len(pro_files), ', '.join(pro_files)))
-            sys.exit(1)
+            print("Found {} .pro files (expected one):\n{}\nChoose one from {}".format(
+                len(pro_files),
+                '\n'.join(["{}. {}".format(index,os.path.basename(item))
+                    for index,item in enumerate(pro_files)]),
+                '({}-{})'.format(0,len(pro_files)-1)))
+            response = sys.stdin.readline().strip()
+            try:
+                pro_file_name = pro_files[int(response)]
+            except:
+                return 1
```

Comment:
Silently terminating is not user friendly. It would be better to display an error message, then loop back to let the user re-enter their choice.